yarn.lock

```diff
@@ -5551,13 +5551,10 @@ levn@~0.3.0:
     prelude-ls "~1.1.2"
     type-check "~0.3.2"
 
-libphonenumber-js@^1.7.26:
-  version "1.7.48"
-  resolved "https://registry.yarnpkg.com/libphonenumber-js/-/libphonenumber-js-1.7.48.tgz#15b7258b642d0ef5faced14401528639be9a8eaa"
-  integrity sha512-qiHOxCBQ8cNqSkpUcAFn0vwk8eZs6Bq6ttQTiiAICEfUZp3uQ/RuOnPqYMw5qciHCrCNjLN3qp3ih1u0+NuzVA==
-  dependencies:
-    minimist "^1.2.5"
```

Comment:
no more dependencies? 😍

Reply:
> no more dependencies? 😍

I think it's one of those cases where the project author mistakenly listed what should have been devDependencies as regular dependencies. At least, I still see minimist defined as a devDependencies of the project ([source](https://gitlab.com/catamphetamine/libphonenumber-js/-/blob/8a706acb0dd3e7da9039d2a8c852159ba9eb71d5/package.json#L23)).

In other words, we probably won't see any direct benefit to runtime bundle size. But from a developer's perspective, we're downloading a bit less when we run `yarn install`.